### PR TITLE
W-16832816-Avro-support-fa

### PR DESCRIPTION
**Pull request by failup**

### CODE REVIEW

File: modules/ROOT/pages/design-async-api.adoc [hunk 1]
```diff
@@ -236,8 +236,90 @@ components:
...
+                    type: string
+                  - name: totalSeats
+                    type: int
+----
+== Example AsyncAPI Specification Fragment (Avro Schema)
+
+If you created an Avro Schema fragment, you can replace the initial API fragment with following example code:
+
+[source,yaml]
+----
```
Comment (lbauret) [inline modules/ROOT/pages/design-async-api.adoc:295]: We may probably add an example on how to reference a fragment avro from an Async API spec. For example for this case:
```
asyncapi: 2.6.0
info:
  title: Example AsyncAPI Spec (Inline Avro)
  version: '1.0.0'
channels:
  myChannel:
    publish:
      message:
        schemaFormat: application/vnd.apache.avro;version=1.9.0
        payload:
          $ref: AmericanFlightDataType.avsc
```

**Review by lbauret — DISMISSED** [review]
LGTM! One suggestion added

File: modules/ROOT/pages/design-async-api.adoc [hunk 2]
```diff
@@ -236,8 +236,90 @@ components:
...
+    publish:
+      message:
+        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+        payload:
+          name: AmericanFlightDataType
+          type: record
+          fields:
+            - name: ID
+              type: int
+              example: 1
```
Comment (BlockLucas) [inline modules/ROOT/pages/design-async-api.adoc]: "example" entry in all these fields are not defined by the spec, after discussing with the team, a good alternative could be "default", although it has a different purpose it fills the console & triggers the validation if the value is incorrect

- name: ID
   type: int
   default: 1

Can you replace them? the same applies for line 264, 267 & so on

File: modules/ROOT/pages/design-async-api.adoc [hunk 3]
```diff
@@ -236,8 +236,90 @@ components:
...
+
+If you created an Avro Schema fragment, you can replace the initial API fragment with following example code:
+
+[source,yaml]
+----
+{
+  "name": "AmericanFlightDataType",
+  "type": "record",
+  "fields": [
+    {"name": "ID", "type": "int", "example": 1},
```
Comment (BlockLucas) [inline modules/ROOT/pages/design-async-api.adoc]: same thing with example here (replace to default)

**Comment by BlockLucas** [comment]
Hi @failup!
I've left some comments regarding the usage of example in the inline & fragment for Avro

File: modules/ROOT/pages/design-async-api.adoc (hunk 1 above)
Comment (llibarona) [reply modules/ROOT/pages/design-async-api.adoc:295]: not sure if this is the right place, but if we mention the file extension `.avsc` we could provide a bit more info, considering it is used by convention and not definition (we internally identify an avro fragment by this extension OR by the exchange.json classifier, but the avro spec makes no mention on how to identify an avro file)

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**